pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -379,9 +387,12 @@ def spawn_ddp_children(self, model):
         os.environ['NODE_RANK'] = node_rank
         os.environ['LOCAL_RANK'] = '0'
 
+        # when user is using hydra find the absolute path
+        path_lib = abspath if not HYDRA_AVAILABLE else to_absolute_path
```

Comment:
can we use `os.path.realpath`?

Reply:
sure, but not instead of hydra.utils.to_absolute_path.
This [this](https://hydra.cc/docs/next/tutorials/basic/running_your_app/working_directory) for context (that api is described at the bottom).